src/commands/create.ts

```diff
@@ -53,13 +54,22 @@ export default class Create extends Command {
 
         const folder = path.join(process.cwd(), info.nameSlug);
 
+        const url =  await cli.prompt(chalk.bold('   What is the server\'s url (include https)?'));
```

Comment:
Are the leading spaces on purpose?

Reply:
Yes it matches indentation with the previous prompts

Reply:
Thanks for explanation!